vars/Suse.yml

```diff
@@ -1,3 +1,40 @@
 ---
-
 # Variables for Suse set in ansible_facts['os_family']
+ha_cluster_pacemaker_shell: crmsh
+
+# Placeholder with pcs name
+__ha_cluster_pcs_provider: crm
+
+__ha_cluster_role_essential_packages:
+  - 'corosync'
+  - 'crmsh'
+  - 'fence-agents'
+  - 'ha-cluster-bootstrap'
+  - 'pacemaker'
+  - 'patterns-ha-ha_sles'
+  - 'resource-agents'
+  - 'cluster-glue'
+  - 'socat'
+  - 'libxml2-tools'
+  - 'ClusterTools2' # not pre-installed in SLES for SAP
+  - 'SAPHanaSR'
+  - 'rsyslog'
```

Comment:
The essential packages variable is supposed to hold only a minimal set of packages which are required by the role to run. For example, when configuring a qnetd node, you don't need pacemaker, corosync, fence-agents, resource-agents, etc. installed there. Those only get installed when `ha_cluster_cluster_present` is `true`.

Reply:
I agree, I will move them under __ha_cluster_fullstack_node_packages instead.

Reply:
I have removed SAPHanaSR, but I am keeping rest in. qnetd setup is not part of this PR so current functionality focuses on cluster, where these packages are required.

Future PR focused on qnet/qdevice should be able to carve out essentials.

Reply:
> Future PR focused on qnet/qdevice should be able to carve out essentials.

Yes.  Let's focus this PR on the changes required to the common role functionality (code that affects both the pcs and the crmsh implementations), then hopefully future changes will only affect code specific to crmsh and will be easier to merge.